packages/core/src/components/_index.scss

```diff
@@ -26,34 +26,78 @@ Styleguide components.usage
 /*
 NPM installation
 
-Install the core package with an NPM client like `npm` or `yarn`, pulling in all relevant dependencies:
+1. Install the core package with an NPM client like `npm` or `yarn`, pulling in all relevant dependencies:
+
+  ```sh
+  npm install --save @blueprintjs/core
+  ```
+
+1. If you see `UNMET PEER DEPENDENCY` errors, you should manually install React:
+
+  ```sh
+  npm install --save react react-dom react-addons-css-transition-group
+  ```
+
+1. After installation, you'll be able to import the React JS components in your application:
+
+  ```
+  // extract specific components
+  import { Intent, Spinner, DatePickerFactory } from "@blueprintjs/core";
+  // or just take everything!
+  import * as Blueprint from "@blueprintjs/core";
+
+  // using JSX:
+  const mySpinner = <Spinner intent={Intent.PRIMARY} />;
+
+  // using the namespace import:
+  const anotherSpinner = <Blueprint.Spinner intent={Blueprint.Intent.PRIMARY}/>;
+
+  // use factories for React.createElement shorthand if you're not using JSX.
+  // every component provides a corresponding <Name>Factory.
+  const myDatePicker = DatePickerFactory();
+  ```
+
+1. Don't forget to include the main CSS file from each Blueprint package! Additionally, the `resources/`
+directory contains supporting media such as fonts and images.
+
+  ```html
+  <!-- in plain old reliable HTML -->
+  <!DOCTYPE HTML>
+  <html>
+    <head>
+      ...
+      <!-- include dependencies manually -->
+      <link href="path/to/node_modules/normalize.css/normalize.css" rel="stylesheet" />
+      <link href="path/to/node_modules/@blueprintjs/core/dist/blueprint.css" rel="stylesheet" />
+      ...
+    </head>
+    ...
+  </html>
+  ```
+
+  ```scss
+  // or, using node-style package resolution in a CSS file:
+  // (dependencies' stylesheets should be resolved automatically)
+  @import "~@blueprintjs/core";
+  ```
 
-```sh
-npm install --save @blueprintjs/core
-```
-
-After installation, you'll be able to import the React components in your application:
+Weight: -1
 
-```
-import { Spinner, DatePickerFactory } from "@blueprintjs/core";
-// or
-import * as Blueprint from "@blueprintjs/core";
+Styleguide components.usage.npm
+*/
 
-// JSX works
-const mySpinner = <Spinner/>;
+/*
+DOM4
 
-// using the namespace import
-const anotherSpinner = <Blueprint.Spinner/>;
+Blueprint relies on a handful of DOM Level 4 API methods: `el.query`, `el.queryAll`, and `el.closest()`.
+We depend on a [polyfill library called `dom4`][dom4] to ensure these methods are always available.
```

Comment:
- "We depend" -> "`@blueprintjs/core` depends"
- "are always available." -> "are available."